### PR TITLE
Release 6.0.0 Pull request

### PR DESCRIPTION
**Pull request by katmsft**

### CODE REVIEW

File: Changelog.txt [hunk 1]
```diff
@@ -1,6 +1,14 @@
 Azure Storage Client Library for C++
 History of Changes
 
+Changes in v6.0.0
+- Upgraded CPPRest to latest version: 2.10.10.
+- Resolved a potential memory leak issue in xml_wrapper.cpp.
+- Resolved an issue where getting range would update the blob/file MD5 unexpectedly.
+- Added an option in CMake to build azure-storage-cpp using /MT
```
Comment (JasonYang-MSFT) [inline Changelog.txt]: It is better to call out the option name here.

File: Changelog.txt [hunk 2]
```diff
@@ -1,6 +1,14 @@
 Azure Storage Client Library for C++
 History of Changes
 
+Changes in v6.0.0
+- Upgraded CPPRest to latest version: 2.10.10.
+- Resolved a potential memory leak issue in xml_wrapper.cpp.
+- Resolved an issue where getting range would update the blob/file MD5 unexpectedly.
```
Comment (JasonYang-MSFT) [inline Changelog.txt]: "getting range" to exactly api name or download range content?

File: Changelog.txt [hunk 3]
```diff
@@ -1,6 +1,14 @@
...
+- Added an option in CMake to build azure-storage-cpp using /MT 
+- Resolved an issue where the first forward slash in the front of the blob name will always be trimmed.
+- Supported cancellation token and millisecond level timeout in C++ client library.
```
Comment (JasonYang-MSFT) [inline Changelog.txt]: There are two changes here.

File: Microsoft.WindowsAzure.Storage/includes/wascore/timer_handler.h [hunk 4]
```diff
@@ -0,0 +1,82 @@
+// -----------------------------------------------------------------------------------------
+// <copyright file="timer_handler.h" company="Microsoft">
+//    Copyright 2018 Microsoft Corporation
```
Comment (JasonYang-MSFT) [inline Microsoft.WindowsAzure.Storage/includes/wascore/timer_handler.h]: 2019?

File: Changelog.txt (hunk 2 above)
Comment (katmsft) [reply Changelog.txt]: Modified the changelog.

File: Changelog.txt (hunk 3 above)
Comment (katmsft) [reply Changelog.txt]: Split into two change logs.

File: Microsoft.WindowsAzure.Storage/includes/wascore/timer_handler.h (hunk 4 above)
Comment (katmsft) [reply Microsoft.WindowsAzure.Storage/includes/wascore/timer_handler.h]: Per offline discussion, this is the time that this file is created, not the time added to Github.

File: Changelog.txt [hunk 5]
```diff
@@ -1,6 +1,14 @@
 Azure Storage Client Library for C++
 History of Changes
 
+Changes in v6.0.0
+- Upgraded CPPRest to latest version: 2.10.10.
+- Resolved a potential memory leak issue in xml_wrapper.cpp.
```
[the remark this answers is not in the document]
Comment (katmsft) [reply Changelog.txt]: Clarified that `potential` means this issue only will happen when calling xml decoding on Linux platforms.

**Pull request merged**